support/azure/azure-kubernetes/availability-performance/container-image-pull-performance.md

```diff
@@ -48,8 +50,32 @@ Switch to newer hardware generation virtual machine (VM) stock keeping units (SK
 
 Increase the available CPU cores on your VM SKU to reduce a CPU bottleneck in image pulling. Allocating more cores helps avoid throttling on the VM.
 
-### Switch back to serialized image pulls
+### Toggle image pull type
+
+Use the following AFEC flag instructions to toggle parallel image on or off. 
+
+When you run az feature list --namespace Microsoft.ContainerService, you should see the following feature appear:
+
+`
+{
+    "id": "/subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/providers/Microsoft.Features/providers/microsoft.ContainerService/features/DisableParallelImagePulls",
+    "name": "microsoft.ContainerService/DisableParallelImagePulls",
+    "properties": {
+      "state": "NotRegistered"
+    },
+    "type": "Microsoft.Features/providers/features"
+}
+`
+
+You can register to the feature as follows:
+
+`az feature register --namespace Microsoft.ContainerService --name DisableParallelImagePulls`
+
+Behavior expectation if a subscription is registered to the feature:
 
-Contact support and request to switch back to serialized image pulls.
+- Upgrading any nodepool or cluster to Kubernetes 1.31 from a version below 1.31 will result in serial image pulling, in contrast to the parallel pulling default in Kubernetes 1.31 and above.
+- Nodepools or clusters already running Kubernetes 1.31, but created before the subscription registration, will initially have parallel image pulling. However, any subsequent node image upgrade or Kubernetes version upgrade will switch the nodepool/cluster to serial image pulling.
+- For any new clusters or nodepools created or upgraded to Kubernetes versions below 1.31, registering for this feature will have no effect as parallel image pulling only applies to versions 1.31 and above.
+- Performing an empty PUT on existing Kubernetes 1.31 nodepools or clusters (created before subscription registration) will not immediately switch parallel image pulling to serial pulling. To trigger the change, the node must undergo a reimage, which only happens during a Kubernetes or node image upgrade. However, the PUT operation will update the flag in the database, and the change will take effect during the next upgrade.
```

Comment:
- Performing an empty PUT on existing Kubernetes 1.31 node pools or clusters (that were created before subscription registration) will not immediately switch parallel image pulling to serial pulling. To trigger the change, the node must undergo a reimage. A reimage occurs only during a Kubernetes or node image upgrade. However, the PUT operation will update the flag in the database, and the change will take effect during the next upgrade.